src/components/destination_list.js

```diff
@@ -20,12 +20,18 @@ const DestinationList = ({
   unselectItems,
   selectedItemRenderer,
   noItemsRenderer,
-  withGrouping
+  withGrouping,
+  enableRightDefault,
+  items
 }) => {
   const SelectionStatusRenderer = selectionStatusRenderer;
   const updatedSelectedItems = withGrouping
     ? groupItems(selectedItems)
     : selectedItems;
+
+  const defaultColumns = enableRightDefault
```

Comment:
What is columns in the react-multi-select context? You're mixing Kenshoo's business logic with the components features.